alb-security-group.tf

```diff
@@ -22,3 +10,18 @@ resource "aws_security_group" "alb" {
   }
   tags = merge(var.tags, var.tags_alb, var.tags_security_group)
 }
+# If migrating from v3.x or lower to v4.x or higher there may be a need to import the alb_ingress rules explicitly
```

Comment:
I think this would be a good comment to move into the changelog breaking section